src/index.js

```diff
@@ -17,6 +17,19 @@ const loadDeps = () => {
 };
 
 module.exports = {
+  config: {
+    defaultConfig: {
+      title: 'Default Config Path',
```

Comment:
I'd add a description clarifying that this is only used if `coffeelint` can't find a configuration on it's own.